lib/index.js

```diff
@@ -54,6 +54,8 @@ SaaSquatch.prototype.identify = function(identify) {
   var paymentProviderId = identify.proxy('traits.paymentProviderId');
   var accountStatus = identify.proxy('traits.accountStatus');
   var referralCode = identify.proxy('traits.referralCode');
+  var userReferralCode = identify.proxy('traits.userReferralCode');
```

Comment:
+1 +1 (both comments) — let's send through as many traits as possible (if they accept arbitrary user attributes) and use `context.SaaSquatch.$` for the `userReferralCode` field. @tsholmes happy to go over why in person when I get back from SEA!